package.json

```diff
@@ -41,8 +42,9 @@
   "scripts": {
     "fix": "eslint . --fix",
     "lint": "eslint .",
-    "smoke-test": "mocha smoke-test/*.js --timeout 5000",
-    "test": "mocha test/*.js"
+    "docs-test": "echo 'add docs test'",
```

Comment:
Let's just install `linkinator` and `jsdoc` and have this set up from the get go